src/lando/main/auth.py

```diff
@@ -1,14 +1,30 @@
+from django.contrib.auth.models import User
 from mozilla_django_oidc.auth import OIDCAuthenticationBackend
 
+from lando.main.models.profile import Profile
+
 
 class LandoOIDCAuthenticationBackend(OIDCAuthenticationBackend):
     """An extended OIDC auth backend that manipulates the user profile."""
 
     def get_userinfo(self, *args, **kwargs):
+        # Store fetched userinfo on this instance.
         self.userinfo = super().get_userinfo(*args, **kwargs)
         return self.userinfo
```

Comment:
I realized that this is not needed, as [`get_or_create_user`](https://github.com/mozilla/mozilla-django-oidc/blob/main/mozilla_django_oidc/auth.py#L358) passes the entirety of `userinfo` as `claims`.